.changeset/great-rules-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/create-mc-app': minor
+---
+
+Added a cloudIdentifier flag for Custom Applications and Custom Views initialisation
```

Comment:
Perhaps we could mention that we are adding the option to the CLI to avoid confusion with it being a new configuration option for custom apps/views
```suggestion
Added a `--cloud-identifier` option to the `create-mc-app` CLI, allowing to specify the [cloudIdentifier](https://docs.commercetools.com/merchant-center-customizations/api-reference/custom-application-config#cloudidentifier) in the customization configuration file.

Example usage:
```bash
npx @commercetools-frontend/create-mc-app@latest \
  my-new-custom-application-project \
  --template starter \
  --cloud-identifier gcp-us
```

```

Reply:
Updated in this commit: [6c34369](https://github.com/commercetools/merchant-center-application-kit/pull/3564/commits/6c343695099ba5e75208ba31fa7ba2dfa63da7ff)